tests/subsys/fs/nvs/src/main.c

```diff
@@ -522,7 +522,7 @@ void test_nvs_full_sector(void)
 	len = nvs_write(&fs, filling_id, &filling_id, sizeof(filling_id));
 	zassert_true(len == sizeof(filling_id), "nvs_write failed: %d", len);
 
-	/* sanitycheck on NVS content */
+	/* twister on NVS content */
```

Comment:
This doesn’t look like a reference to Zephyr’s sanitycheck script rather the general concept of a sanity check

Reply:
Removed that change.